.github/workflows/auto-release.yml

```diff
@@ -13,4 +13,38 @@ jobs:
     name: Automated release
     runs-on: ubuntu-latest
     steps:
-      - run: echo "Triggered successfully"
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          # Ensure we are checked out on the develop branch
+          ref: develop
+          # Ensure custom credentials are used when pushing
+          persist-credentials: false
+          # Fetch everything so we can checkout master
+          fetch-depth: 0
+      - name: Update simple-icons and package version
+        id: update
+        run: node scripts/version.js
+      - name: Commit updates
+        run: |
+          # Set up git credential
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Actions"
+
+          # Create a version bump commit
+          git add .
+          git commit -m "Version bump"
+      - name: Merge develop into master
+        run: |
+          git checkout master
+          git merge develop --no-ff -m "Release ${{ steps.update.outputs.NEW_VERSION }}"
+      - name: Push version bump
+        run: |
+          git remote remove origin
+          git remote add origin https://${{secrets.RELEASE_TOKEN}}@github.com/ericcornelissen/to-repo.git
```

Comment:
This sets up the remote so that pushes use a Personal Access Token configured in this repository (needed so that [the Publish workflow](https://github.com/simple-icons/simple-icons-font/blob/5d9d06cf6e2ba4e2bc09fbb81a25b5760a5e5873/.github/workflows/publish.yml) runs on `master`).

> I'm not sure what is happening here with `ericcornelissen/to-repo` origin :thinking: 

Oops, that is the name of the private test repository I used 😅